src/classes/MyHandler/MyHandler.ts

```diff
@@ -1882,6 +1916,15 @@ export default class MyHandler extends Handler {
                 highValue: isContainsHighValue ? highValueMeta : undefined
             };
 
+            // don't use business logic if the bot is not operational
+            if (this.bot.isHalted) {
+                return {
+                    action: 'skip',
+                    reason: 'REVIEW',
+                    meta: meta
+                };
+            }
+
```

Comment:
Oh, shoot...  I thought this one was here
https://github.com/TF2Autobot/tf2autobot/blob/55aea3072c8c295adb9755840c5fb39d16b7bd21/src/classes/MyHandler/MyHandler.ts#L2056-L2060

It's completely newly added. My bad. 😅😅😅